build.gradle

```diff
@@ -288,7 +288,7 @@ subprojects {
         if (dep.group == null || false == dep.group.startsWith('org.elasticsearch')) {
           return
         }
-        Project upstreamProject = dependencyToProject(dep)
+        Project upstreamProject = project.ext.dependencyToProject(dep)
```

Comment:
`project.ext` is a generic extension manager. 
Gradle looks things up there when you reference it on project. 
I don't like ti as it makes it too easy to create coupling throughout the build and makes it hard to see where things are coming from.  That's why I  prefer to use `project.ext` to be explicit. In this case it's required because Gradle looks somewhere else for `dependencyToProject` and does not find it. I didn't go into more detail when making this change.